packages/@uppy/dashboard/src/components/FileCard/index.scss

```diff
@@ -1,3 +1,6 @@
+@use '../../../../core/src/_variables.scss' as *;
```

Comment:
is this to fix the warnings? is it not possible to instead write something like `@use '@uppy/core/src/_variables.scss' as *;`?

Reply:
Not possible unfortunately. I has to rewrite these imports by hand or sass crashed.